### ISSUE TITLE
False positive 207 when using attempting to embed double-curlies in output

### ISSUE THREAD

**Issue by lucastheisen** [ansible-community/ansible-lint#1112]
##### Summary
We have a play that prints a message to the user using pause that indicates a command to copy/paste which includes `{{` and `}}`.  `ansible-lint` sees this as a nested jinja pattern.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
# ansible --version
ansible 2.10.3
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/ltheisen/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ltheisen/.ansible-venv/lib/python3.6/site-packages/ansible
  executable location = /home/ltheisen/.ansible-venv/bin/ansible
  python version = 3.6.9 (default, Nov  7 2019, 10:44:02) [GCC 8.3.0]

# ansible-lint --version
ansible-lint 4.3.7
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

WSL running:
```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic
```

##### STEPS TO REPRODUCE

```yaml
---
- hosts: localhost
  gather_facts: no

  tasks:

  - name: Provide user manual process instructions
    pause:
      prompt: |
        Please run the following command to copy the image (press enter when complete):
            docker image inspect {{ image }} --format='{{'{{'}}.Size{{'}}'}}'
    vars:
      image: my-app
```

##### Desired Behaviour

Should not report nested jinja.

##### Actual Behaviour

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:
- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```console
DEBUG    Logging initialized to level 10
DEBUG    Options: Namespace(colored=True, config_file=None, display_relative_path=True, exclude_paths=[], format='rich', listrules=False, listtags=False, parseable=False, parseable_severity=False, playbook=['/tmp/ansiblelintbug.yml'], progressive=False, quiet=False, rulesdir=[], skip_list=[], tags=[], use_default_rules=False, verbosity=3, warn_list=[])
DEBUG    Loading rules from /home/ltheisen/.ansible-venv/lib/python3.6/site-packages/ansiblelint/rules
DEBUG    Examining ../../../../../../tmp/ansiblelintbug.yml of type playbook
WARNING  Listing 1 violation(s) that are fatal
[207] Nested jinja pattern
../../../../../../tmp/ansiblelintbug.yml:7
Task/Handler: Provide user manual process instructions

You can skip specific rules or tags by adding them to your configuration file:

┌────────────────────────────────────────────────────────────────────────────────┐
│ # .ansible-lint                                                                │
│ warn_list:  # or 'skip_list' to silence them completely                        │
│   - '207'  # Nested jinja pattern                                              │
└────────────────────────────────────────────────────────────────────────────────┘
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
